MFVNeutralino/plugins/VertexTracks.cc

```diff
@@ -687,14 +683,15 @@ bool MFVVertexTracks::filter(edm::Event& event, const edm::EventSetup& setup) {
     const int nsthits = tk->hitPattern().numberOfValidStripHits();
     const int npxlayers = tk->hitPattern().pixelLayersWithMeasurement();
     const int nstlayers = tk->hitPattern().stripLayersWithMeasurement();
-    const auto trackLostInnerHits = tk->hitPattern().numberOfLostHits(reco::HitPattern::MISSING_INNER_HITS);
+    const double track_recon_eff = 1-1.11786*fabs(dxybs)*fabs(dxybs); //Alec added
+    if (r3->Uniform(0,1) > track_recon_eff) continue; //Alec added
```

Comment:
@duquale this throwing away of tracks was done for a study, but do we actually want to be applying this in our code at all times? I think this may even apply it to data! Let's revert this.